SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialPackageMapClient.h

```diff
@@ -62,6 +62,10 @@ class SPATIALGDK_API USpatialPackageMapClient : public UPackageMapClient
 
 	virtual bool SerializeObject(FArchive& Ar, UClass* InClass, UObject*& Obj, FNetworkGUID *OutNetGUID = NULL) override;
 
+	// Pending object references, being asynchronously loaded.
+	typedef TSet<FNetworkGUID> PendingRefSet;
+	PendingRefSet PendingReferences;
```

Comment:
Should this be a public field or a function that adds things to it. It seems like things are only added externally and all reading and removing comes from internal stuff.

Reply:
I had a thought about making it private, but : 
1) I need to do the IsPendingGUID check in FUnrealObjectRef::ToObjectPtr anyway, to set bOutUnresolved
2) I need to browse and remove elements from this set somewhere where the receiver is available.

We could devise a set of methods to achieve a similar result and proper encapsulation (extra out unresolved parameter, template browse method to keep the packagemap independent from the receiver, etc...), but this is as straightforward as it gets.